### ISSUE TITLE
Complete CONTRIBUTING.md

### ISSUE THREAD

**Issue by TorrentofShame** [KnightHacks/hackathon-2021-backend#49]
The `CONTRIBUTING.md` file is currently blank, it should contain information.

**Comment by APherwani** [comment]
I think this is part of a larger conversation we need to have in the dev team. How do we want to support external contributors?

**Comment by TorrentofShame** [comment]
> I think this is part of a larger conversation we need to have in the dev team. How do we want to support external contributors?

I think we would support external contributors in the same way other open source projects do, they can submit issues and prs, and someone on the team will review, etc.

The most important part of this file is to make it easy for new backend team members to get started.

**Comment by APherwani** [comment]
I don't have a problem with it but I want to get the opinion of other leads.
@aileenpongnon @ElbertTse @AdamPellot @suneettipirneni

**Comment by suneettipirneni** [comment]
I'm not sure if our input is valid in this situation. However as long as licensing is open, I think it's up to the maintainer's discretion as to what their contributing file should contain.

**Comment by APherwani** [comment]
What we decide here should apply to all other open source repos from knighthacks so I think your input is important.

**Comment by aileenpongnon** [comment]
> I don't have a problem with it but I want to get the opinion of other leads. @aileenpongnon @ElbertTse @AdamPellot @suneettipirneni

disagree

**Comment by TorrentofShame** [comment]
@APherwani contributing.md is for how to contribute to the repo, not who and what and can.

It just says how to create/submit issues and prs.

**Comment by TorrentofShame** [comment]
@APherwani No, nor is this thhe place for it.

**Comment by APherwani** [comment]
Fair point

**Comment by APherwani** [comment]
I agree but Simon has a point. This is the wrong place to talk about that. Sorry guys this is my fault for starting the conversation here in the first place.
